router/internal/requestlogger/access_log_expression.go

```diff
@@ -0,0 +1,32 @@
+package requestlogger
```

Comment:
The other function that tests the logger lies in this package `requestlogger`, but creating of the requestContext which is needed for evaluating expressions is in the `core` package, and *is not* exported.

Which means if I was to write my test cases the same way here in the `subgraph_logger_test.go` file, I will need to export the `buildRequestContext` function (and related struct), which seemed like a bad idea for this one test.

But it also made me wonder if that code should be in a separate (maybe "inner-core" package, or this package merged with core since its quite small?). I ended up writing a unit test directly `access_log_field_handler_test` in the core package.